src/git-review.zig

```diff
@@ -194,6 +199,75 @@ fn review_lgtm(shell: *Shell) !void {
     try shell.exec("git push --force-with-lease", .{});
 }
 
+fn review_split(shell: *Shell) !void {
+    if (!try has_review(shell)) {
+        log.err("no review in progress", .{});
+        return error.NoReview;
+    }
+    const commit = try shell.exec_stdout("git rev-parse HEAD", .{});
```

Comment:
Before we start making any changes to the git tree, can we print this commit hash to `log.info`? That way if there's some kind of problem with the script, you can always easily restore the original state.

Maybe even print something like `To undo the split, run "git reset ..."`.

Reply:
Yeah, that's exactly how it works, see `errdefer` on line 223. Operations before that are read-only.

Reply:
Oh, are you talking about printing this unconditionally? Yeah, that's probably better: even if the script _thinks_ it worked correctly, it might be buggy! I actually originally just had an info log there, let me restore it back.

Reply:
Yeah, even if there isn't a zig error, we might still not like the result